jenkins/test-boot1

```diff
@@ -26,7 +26,11 @@ HARE_DIR=/opt/seagate/cortx/hare
 export PATH="$HARE_DIR/bin:$PATH"
 
 do_io() {
-    $HARE_DIR/libexec/m0crate-io-conf > _m0crate-io.yaml
+    if [ "$( echo $( grep 'transport_type' $confstore ) | awk -F": " '{print $2}' )" = "lnet" ]; then
```

Comment:
@mssawant @suvratjoshi 
Added Changes as per [PR#1956](https://github.com/Seagate/cortx-hare/pull/1956)

Reply:
Looks like, premerge for this PR is still failing